build.gradle

```diff
@@ -1,8 +1,8 @@
 buildscript {
   ext.versions = [
       'minSdk': 9,
-      'compileSdk': 25,
-      'buildTools': '26.0.2',
+      'compileSdk': 27,
+      'buildTools': '27.0.3',
```

Comment:
You can delete this property and all references to it. AGP will install what it needs itself.

Reply:
The main goal of my PR is to update this
`ddmlib': 'com.android.tools.ddms:ddmlib:26.0.1`
Ok, I update some more libs, but for me it's hard to remove some variables, which I didn't introduced, and it's usages, which the inventor doesn't like anymore.

It's more effective if you change it, you know exactly what you want and how should it look like